package.json

```diff
@@ -9,7 +9,7 @@
     "url": "git@github.com:bcoe/c8.git"
   },
   "scripts": {
-    "test": "node ./bin/c8.js --reporter=html --reporter=text mocha ./test/*.js",
+    "test": "node ./bin/c8.js --reporter=html --reporter=text mocha --timeout=4000 ./test/*.js",
```

Comment:
Is the extra timeout just for windows?

Not to be a stickler, but, one of the really nice differentiators of this library is that it runs through tests much faster than NYC, which in turn makes development and contributions a lot easier. I would love to be in a world where that stays true as much as possible

Reply:
I had the Node.js test suite and c8 test suite running at the same time, which was slowing down my computer. Reverted this change.